Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>When you find yourself adding string columns - ask yourself what you are doing wrong and how can you simplify.</p><p>Here, simply add two columns for the points.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>